cli/src/commands/next.rs

```diff
@@ -56,9 +56,15 @@ pub(crate) struct NextArgs {
     offset: u64,
     /// Instead of creating a new working-copy commit on top of the target
     /// commit (like `jj new`), edit the target commit directly (like `jj
-    /// edit`).
-    #[arg(long, short)]
+    /// edit`). Takes precedence over config in `ui.movement.edit`; i.e.
+    /// will negate `ui.movement.edit = "never"`
+    #[arg(long, short, conflicts_with = "no_edit")]
```

Comment:
nit: remove `, conflicts_with = "no_edit"`?

It's implied by `no_edit`'s attribute, and we don't have the one in `PrevArgs`.

Reply:
Done. The `conflicts_with` attribute is only set on `no_edit` now, in both {next,prev}.rs.